Documentation/security/network/encryption-ipsec.rst

```diff
@@ -182,6 +182,11 @@ included and should be monotonically increasing every re-key with a rollover
 from 15 to 1. The Cilium agent will default to ``KEYID`` of zero if its not
 specified in the secret.
 
+If you are using Cluster Mesh, the key rotation procedure needs to be applied
+to all clusters in the mesh. You might need to adjust the transition time to
```

Comment:
How about make it more specific, `adjust` -> `increase`?

Reply:
Done, thanks.